dashboard_data/CombineMultipleLayers(SerialChart).md

```diff
@@ -30,51 +30,53 @@ var janssen = GroupBy(
     [{ name: 'janssen', expression: 'F_1st_dose_allocations', statistic: 'SUM' }]
 );
 
-var combinedDict = {
-    fields: [
-        { name: 'manufacturer', type: 'esriFieldTypeString' },
-        { name: 'week_of_allocation', type: 'esriFieldTypeString' },
-        { name: 'count_of_doses', type: 'esriFieldTypeInteger' },
-    ],
-    geometryType: '',
-    features: [],
-};
+// Create empty array for features
+var features = [];
 
-// Loop through each of the three FeatureSets and store attributes into a combined dictionary.
-var i = 0;
+// Loop through each of the three FeatureSets and populate feature array.
 for (var m in moderna) {
-    combinedDict.features[i] = {
+    var feat = {
         attributes: {
             manufacturer: 'Moderna',
             week_of_allocation: m['week_of_allocations'],
             count_of_doses: SUM(m['moderna_1'], m['moderna_2']),
         },
     };
-    i++;
+    Push(features, feat);
 }
 
 for (var p in pfizer) {
-    combinedDict.features[i] = {
+    var feat = {
         attributes: {
             manufacturer: 'Pfizer',
             week_of_allocation: p['week_of_allocations'],
             count_of_doses: SUM(p['pfizer_1'], p['pfizer_2']),
         },
     };
-    i++;
+    Push(features, feat);
 }
 
 for (var j in janssen) {
-    combinedDict.features[i] = {
+    var feat = {
```

Comment:
could declare feat outside the loop.  Since the variable is being reused, we do not need to declare it again.  Same comment in the other loops